dataclasses_json/cfg.py

```diff
@@ -41,30 +39,32 @@ def __init__(self):
 global_config = _GlobalConfig()
 
 
-def config(metadata: dict = None, *,
-           # TODO: these can be typed more precisely
-           # Specifically, a Callable[A, B], where `B` is bound as a JSON type
-           encoder: Callable = None,
-           decoder: Callable = None,
-           mm_field: MarshmallowField = None,
-           letter_case: Callable[[str], str] = None,
-           undefined: Optional[Union[str, Undefined]] = None,
-           field_name: str = None,
-           exclude: Optional[Callable[[str, T], bool]] = None,
-           ) -> Dict[str, dict]:
+def config(
+    metadata: dict = None,
+    *,
+    # TODO: these can be typed more precisely
+    # Specifically, a Callable[A, B], where `B` is bound as a JSON type
+    encoder: Callable = None,
+    decoder: Callable = None,
+    mm_field: MarshmallowField = None,
+    letter_case: Union[Callable[[str], str], 'LetterCase'] = None,
+    undefined: Optional[Union[str, Undefined]] = None,
+    field_name: str = None,
+    exclude: Optional[Callable[[str, T], bool]] = None,
+) -> Dict[str, dict]:
     if metadata is None:
         metadata = {}
 
-    lib_metadata = metadata.setdefault('dataclasses_json', {})
+    lib_metadata = metadata.setdefault("dataclasses_json", {})
 
     if encoder is not None:
-        lib_metadata['encoder'] = encoder
+        lib_metadata["encoder"] = encoder
 
     if decoder is not None:
-        lib_metadata['decoder'] = decoder
+        lib_metadata["decoder"] = decoder
 
     if mm_field is not None:
-        lib_metadata['mm_field'] = mm_field
+        lib_metadata["mm_field"] = mm_field
```

Comment:
Here and below: why do you change the single quotes for the double ones?

Reply:
I think it was because I formatted w/ black.